### PR TITLE
feat(Icon): Add and export missing icons to Teams Theme 

### PR DESCRIPTION
**Pull request by joheredi**
Adding and exporting the following icons
image

The 3 added icons do not contain an outline version. Spoke with @codepretty offline about this and she is requesting the outline versions from design, but suggested to add the filled version and update when the outlines are ready.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add `expanded` prop to `Accordion` @silviuavram ([#1322](https://github.com/stardust-ui/react/pull/1322))
 - Replace `react-popper` package with custom `Popper` component and exposed as `UNSTABLE_Popper` positioning helper @Bugaa92 ([#1358](https://github.com/stardust-ui/react/pull/1358))
 - Add strict types for the `colorScheme` props used in the components' `variables` @mnajdova([#1340](https://github.com/stardust-ui/react/pull/1340))
+- Add and Export message-seen, presence-available, presence-stroke and open-outside  icons in Teams Theme @joheredi ([#1390](https://github.com/stardust-ui/react/pull/1390))
```
Comment (mnajdova) [inline CHANGELOG.md]: Please mention the eye-friendlier icon as well.

**Review by mnajdova — APPROVED** [review]

**Comment by kuzhelov** [comment]
@codepretty, could you, please, check the names for exported icons?

- `message-seen`
- `presence-available`
- `presence-stroke`
- `open-outside` 
- `eye-friendlier`

File: CHANGELOG.md (hunk 1 above)
Comment (joheredi) [reply CHANGELOG.md]: good catch, thanks!

**Comment by codepretty** [comment]
a lot of the names are too context specific. would suggest renaming to something more descriptive of what the icon looks like or more generic cases.
Some examples & what i found on fontawesome:
`message-seen` should be something similar to `eye-` like `eye-friendlier`
`presence-available` - `check-circle`
`presence-stroke` - `circle`
`open-outside` - `external-link`

**Pull request merged**